test/ArrtistryTest.js

```diff
@@ -0,0 +1,19 @@
+const { TestHelper } = require('@openzeppelin/cli');
+const { Contracts, ZWeb3 } = require('@openzeppelin/upgrades');
+
+ZWeb3.initialize(web3.currentProvider);
+
+const Sample = Contracts.getFromLocal('Arrtistry');
+
+require('chai').should();
+
+contract('Sample', function () {
+
+  beforeEach(async function () {
+    this.project = await TestHelper();
+  })
+
+  it('should create a proxy', async function () {
+    const proxy = await this.project.createProxy(Sample);
```

Comment:
Probably not a sufficient contract. Probably best to mock up a Contract and check that method calls are passed down e.g. call proxy.foo() and Sample.foo should return a magic number

Reply:
What I have done is create a new contract Proxyable.sol which comes along with a function we can test and written a test for that to make sure we can Proxyable that contract. I have removed functionality from the ArrtistryTest.js since not really any functionality to test in that yet